src/ops.rs

```diff
@@ -42,6 +44,7 @@ impl From<Config> for Ops {
                 .client_cleanup_interval_s
                 .map(Duration::from_secs)
                 .unwrap_or(def_client_cleanup_interval),
+            metrics: Metrics::new(),
```

Comment:
Does not need to be part of the state and can be retrieved stateless on API query (which is already done)

Reply:
replied before